php-classes/ReCaptcha.php

```diff
@@ -0,0 +1,2246 @@
+<?php
+    
+/*
+Google reCAPTCHA class for PHP
+Author: Tribulant Software
+Author URL: http://tribulant.com
+Version: 1.0	
+*/
+
+/*
+ * http.php
+ *
+ * @(#) $Header: /opt2/ena/metal/http/http.php,v 1.92 2014/08/14 23:17:34 mlemos Exp $
+ *
+ */
+
+define('HTTP_CLIENT_ERROR_UNSPECIFIED_ERROR',       -1);
+define('HTTP_CLIENT_ERROR_NO_ERROR',                 0);
+define('HTTP_CLIENT_ERROR_INVALID_SERVER_ADDRESS',   1);
+define('HTTP_CLIENT_ERROR_CANNOT_CONNECT',           2);
+define('HTTP_CLIENT_ERROR_COMMUNICATION_FAILURE',    3);
+define('HTTP_CLIENT_ERROR_CANNOT_ACCESS_LOCAL_FILE', 4);
+define('HTTP_CLIENT_ERROR_PROTOCOL_FAILURE',         5);
+define('HTTP_CLIENT_ERROR_INVALID_PARAMETERS',       6);
+
+class http_class

```

Comment:
@RowdyChildren It's not ok to declare additional classes in a php-classes file, each class should define exactly one class - the one it's named for. Break out http_class and its constants to its own file. Ideally it would be in a namespace too given how generic the name is. I'd create a folder/namespace called Tribulant to reflect the vendor and put them both in there